deposits.md

```diff
@@ -0,0 +1,225 @@
+# Deposits
+
+Deposits are transactions which are initiated on L1, and executed on L2. This document outlines a new
+[Transaction Type][transaction-type] for deposits. It also describes how deposits are initiated on
+L1, along with the authorization and validation conditions on L2.
+
+## The Deposit Transaction Type
+
+[deposit-transaction-type]: #deposit-transaction-type
+
+Deposit transactions have the following notable distinctions from existing transaction types:
+
+1. They are derived from Layer 1 blocks, and must be included as part of the protocol.
+2. They do not include signature validation (see [L1 Transaction deposits][#l1-transaction-deposits] for the rationale).
+
+We define a new [EIP-2718] compatible transaction type with the prefix `0x7E`, and the following
+fields:
+
+- `address to`
+- `address from`
+- `uint256 value`
+- `bytes data`
+- `uint256 gasLimit`
+
+This is a subset of the fields used in [EIP-155], but does not include signature information.
+
+We select `0x7E` because transaction type identifiers are currently allowed to go up to `0x7F`.
+Picking a high identifier minimizes the risk that the identifier will be used by Ethereum in the
```

Comment:
I think this should be reworded to maintain the meme that OP is Ethereum. We should also consider opening an EIP for this once its standardized

Reply:
Do you have a suggestion on the rewording?

Reply:
Good callout @tynes. For now I'll go with this: 
> Picking a high identifier minimizes the risk that the identifier will be used be claimed by another
> transaction type on the L1 chain in the future.... 
And hard agree that we should open an EIP as soon as we're ready.